client/src/main/java/io/pravega/client/batch/StreamInfo.java

```diff
@@ -10,17 +10,37 @@
 package io.pravega.client.batch;
 
 import com.google.common.annotations.Beta;
-import io.pravega.client.stream.Stream;
+import io.pravega.client.stream.StreamCut;
 import lombok.Data;
 
+/**
+ * This class is used to represent Stream information. It currently includes:
+ *
+ *  <li> - scope of stream.
+ *  <li> - name of stream.
+ *  <li> - {@link StreamCut} which represents the current TAIL of the stream.
+ *  <li> - {@link StreamCut} which represents the current HEAD of the stream.
+ */
 @Beta
 @Data
 public class StreamInfo {
```

Comment:
We still have creationTime and lastUpdated time that we could be returning. Is that going to be in a separate PR?

Reply:
yes, will raise it as a separate PR.